Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -1088,7 +1090,7 @@ private extension PurchasesOrchestrator {
             if let completion = self.getAndRemovePurchaseCompletedCallback(forTransaction: purchasedTransaction) {
                 self.operationDispatcher.dispatchOnMainActor {
                     completion(purchasedTransaction,
-                               result.value,
```

Comment:
There's a possibility that this ends up with `nil` customer info and `nil` error, leading to a crash: https://app.circleci.com/pipelines/github/RevenueCat/purchases-ios/13951/workflows/3b380d52-bc83-4449-aa91-cf6b55235ec8/jobs/103818

Need to write a unit test for this and fix it.

Reply:
Fixed.